hw/ip/entropy_src/doc/theory_of_operation.md

```diff
@@ -148,13 +148,13 @@ However, if the ENTROPY_SRC block experiences internal back pressure, health tes
 The reduce the probability of dropping post-health test entropy bits, the **Distribution FIFO** can be used.
 This FIFO has pass-through mode enabled meaning it doesn't add latency to hardware pipeline.
 It has a width of 32 bits.
-Its depth is configurable via compile-time Verilog parameter and should matched to the expected level of conditioner back pressure.
+Its depth is configurable via compile-time Verilog parameter and should be matched to the expected level of conditioner back pressure.
 The level of conditioner back pressure depends on the following factors:
-- The maximum latency it takes for the conditioner to add the padding bits \\(n_{pad}\\) (25 clock cycles) and to run the internal SHA3 primitive \\(n_{sha3}\\) (24 clock cycles).
+- The maximum latency it takes for the conditioner to add the padding bits \\(n_{pad}\\) (25 clock cycles) and to run the internal SHA3 primitive \\(n_{SHA3}\\) (24 clock cycles).
```

Comment:
The capitals here and on line 157 don't look right to me. I think we want to keep lower case for a variable name?

Reply:
Maybe we also want to remove `it takes` from this sentence.

Reply:
The intention was to keep acronyms (SHA3, PTRNG) in all caps and the rest all lower case. But it doesn't look nice. I've now switched to all lower case for consistency.